src/metabase/driver/sql_jdbc/execute.clj

```diff
@@ -445,7 +445,7 @@
   (when (< (try (.. ^PreparedStatement stmt getParameterMetaData getParameterCount)
                 (catch Throwable _ (count params)))
            (count params))
-    (throw (ex-info (tru "It looks like we got more parameters than we can handle, remember that parameters cannot be used in comments or as identifiers.")
```

Comment:
I believe the original message is correct here; it's not about _having_ more parameters, it's about having _gotten_ more parameters.